GROMACS.md

```diff
@@ -0,0 +1,186 @@
+# GROMACS 2021.2 Porting Guide(CentOS 7.6)
+
+## Introduction
+
+## Environment Requirements
+
+### Software Requirements
+
+| Item     | Version | Download Address                                             |
+| -------- | ------- | ------------------------------------------------------------ |
+| FFTW3    | 3.3.9   | https://www.fftw.org/fftw-3.3.9.tar.gz                       |
+| OpenBLAS | 0.3.15  | https://github.com/xianyi/OpenBLAS/releases/download/v0.3.15/OpenBLAS-0.3.15.tar.gz |
+| CMake    | 3.21.1  | https://github.com/Kitware/CMake/releases/download/v3.21.1/cmake-3.21.1.tar.gz |
+| GROMACS  | 2021.2  | https://ftp.gromacs.org/gromacs/gromacs-2021.2.tar.gz        |
+
+### OS Requirements
+
+| Item    | Version    | How to Obtain                    |
+| ------- | ---------- | -------------------------------- |
+| Cent OS | 7.6        | https://www.centos.org/download/ |
+| Kernel  | 4.14.0-115 | Included in the OS image         |
+
+## Configuring the Compilation Environment
+
+### Installing dependencies
+
+
+```shell
+yum install time -y
+yum install curl* -y
+yum install csh -y
+yum install zlib* -y
+```
+
+### Installing GNU 9.3
+
+
+```shell
+yum install -y centos-release-scl
+yum install -y devtoolset-9-gcc
+yum install -y devtoolset-9-gcc-c++
+yum install -y devtoolset-9-binutils
+scl enable devtoolset-9 bash
+echo "souce /opt/rh/devtoolset-9/enable" >> /etc/profile
+```
+
+### Installing Open MPI
+
+1. Install the system dependency package:
+
+
+```shell
+yum install libxml2* systemd-devel.aarch64 numa* -y
+```
+
+2. Install Open MPI:
```

Comment:
Should Open MPI be added to the table with software dependencies above ?

Reply:
Yes, we should and modified.